dashboard/templates/dashboard/pset_queue_list.html

```diff
@@ -22,7 +22,11 @@
             {% if pset.student.user == request.user or request.user.is_staff %}
               <a href="{{ pset.get_absolute_url }}">{{ pset.unit.group.name }}<sup>{{ pset.unit.code }}</sup></a>
             {% else %}
-              {{ pset.unit.group.name }}<sup>{{ pset.unit.code }}</sup>
+              {% if pset.unit.code == 'BKV' or pset.unit.code == 'DKV' or pset.unit.code == 'ZKV' %}
```

Comment:
Surely there's a better check possible?

Reply:
This seemed the easiest to me as I was not sure of what other attributes the pset type object had.

Reply:
@RadialFunc `core/models.py` is what you want if you want to see what other attributes there are.

In this case you want `pset.unit.group.hidden`